### PR TITLE
Adjust log fields

### PR DESCRIPTION
**Pull request by mohabusama**
- use app & version.
- add container_name.
- use appdynamics_app & appdynamics_tier labels.
- override existing job files if it is the first run.

### CODE REVIEW

File: k8s_log_watcher/agents/appdynamics.py [hunk 1]
```diff
@@ -9,6 +9,6 @@ def get_template_path():
 
 def get_template_vars(pod_name, pod_labels):
     return {
-        'app_name': pod_labels.get('app_name'),
-        'app_tier': pod_labels.get('app_tier'),
+        'app_name': pod_labels.get('appdynamics_app'),
+        'app_tier': pod_labels.get('appdynamics_tier'),
```
Comment (mohabusama) [inline k8s_log_watcher/agents/appdynamics.py]: As I understood, we won't use namespaces, and use ``app`` & ``version`` directly (as prefixes are optional). I am also kind of wary of Z specific key words slipping into opensource projects. Is there a label prefix we use elsewhere?

File: k8s_log_watcher/main.py [hunk 2]
```diff
@@ -126,22 +140,24 @@ def sync_containers_job_files(containers, containers_path, dest_path, kube_url=N
...
+            kwargs['app_version'] = pod_labels.get('version')
             kwargs['pod_name'] = pod_name
+            kwargs['container_name'] = container_name
             kwargs['node_name'] = CLUSTER_NODE_NAME
 
             if not all([kwargs['app_id'], kwargs['app_version']]):
                 logger.warning(
-                    ('Labels "app_id" and "app_version" are required for container({}) in pod({})'
-                     ' ... Skipping!').format(container['id'], pod_name))
+                    ('Labels "app_id" and "app_version" are required for container({}: {}) in pod({})'
```
Comment (mohabusama) [inline k8s_log_watcher/main.py]: Fixing.

**Comment by mohabusama** [comment]
👍

**Comment by Jan-M** [comment]
:+1:

**Pull request merged**